app/src/main/java/org/p2p/wallet/home/ui/main/HomeElementItemMapper.kt

```diff
@@ -26,9 +26,9 @@ class HomeElementItemMapper(
 
         val result = mutableListOf<HomeElementItem>(HomeElementItem.Title(R.string.home_tokens))
 
-        result += ethereumTokens.map {
+        result += ethereumTokens.map { it ->
             val claimStatus = ethereumBundleStatuses[it.publicKey]
-            HomeElementItem.Claim(it, isClaimEnabled = claimStatus?.canBeClaimed() ?: true)
+            HomeElementItem.Claim(it, isClaimEnabled = claimStatus?.all { it?.canBeClaimed() == true } ?: true)
```

Comment:
claimStatus?.all { it?.canBeClaimed() == true } ?: true better to be extracted to variable